src/main/kotlin/org/openbroker/no/model/BankAccount.kt

```diff
@@ -0,0 +1,33 @@
+package org.openbroker.no.model
+
+private val clearingNumberRegex = Regex("^[1-9][0-9]{3}\$")
```

Comment:
You dont need backslack for end of line symbol. So it should be `^[1-9][0-9]{3}$`

Reply:
Thanks, this has been updated.